<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qBittorrentConfPanel</class>
 <widget class="QWidget" name="qBittorrentConfPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Co&amp;nfig Directory:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>configDirPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="configDirPathEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseConfigDir">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Bac&amp;kup Directory:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>backupDirPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="backupDirPathEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="browseBackupDir">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseConfigDir</sender>
   <signal>clicked()</signal>
   <receiver>qBittorrentConfPanel</receiver>
   <slot>browseConfigDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseBackupDir</sender>
   <signal>clicked()</signal>
   <receiver>qBittorrentConfPanel</receiver>
   <slot>browseBackupDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseConfigDir()</slot>
  <slot>browseBackupDir()</slot>
 </slots>
</ui>
